applications/tari_collectibles/src-tauri/src/clients/wallet_client.rs

```diff
@@ -55,6 +55,15 @@ impl WalletClient {
     Ok(())
   }
 
+  fn get_inner(
```

Comment:
nit: convention when returning mutable references
```suggestion
  fn get_inner_mut(
```

Reply:
Thanks 💯